### PR TITLE
Scorecard and guide

### PR DESCRIPTION
**Pull request by landonreed**
This PR:
- adds the framework for the scorecard (screenshot 1) - note: there's a little bit of choppiness on expanding a div (any help is welcome!)
- adds the Scoring Guide popup (screenshot 2)

Once the back end is connected, we should replace the mock data for the score card with the actual category data.

### Scorecard
image

### Scoring Guide
image

### CODE REVIEW

File: components/Scorecard.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                        <FontAwesomeIcon className='fa-2x' icon={ faCaretDown } />
+                    </div>
+                </h2>
+            </div>
+            <div id={collapseId}
+                className="collapse"
+                aria-labelledby={headingId}
+                data-parent="#scorecard"
+            >
+                <div className="p-0 pb-3 card-body small font-italic">
```
Comment (egiurleo) [inline components/Scorecard.js:54]: I moved the padding (`pb-3`) from the collapse element to the child and it fixed the choppiness! Let me know if this seems like an okay thing to do.

File: components/common/ScoringGuide.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        text="Scoring guide"
+        title="Scoring guide">
+        <div className="container-fluid">
+            <div className="row">
+                <h4 className="guide-heading">Overall State Score</h4>
+                <div className='overall scoring-guide'>
+                    {[4, 3, 2, 1, 0].map(score => (
+                        <ScoreDescription key={score} score={score}/>
+                    ))}
+                </div>
```
Comment (egiurleo) [inline components/common/ScoringGuide.js:27]: For some reason the positioning of the overall scores are messed up on Chrome on my machine. Is this just me?

Screen Shot 2021-02-21 at 6 18 02 PM

**Review by egiurleo — COMMENTED** [review]
This looks awesome! I'm happy to approve once the "overall" category in the scoring guide is fixed.

File: components/common/ScoringGuide.js (hunk 2 above)
Comment (landonreed) [reply components/common/ScoringGuide.js:27]: Yikes! I wonder what happened there. I'll take a look.

Comment (landonreed) [reply components/common/ScoringGuide.js:27]: Bad css selector. Will fix.

**Review by egiurleo — APPROVED** [review]
Yay! This is awesome 🎉

**Pull request merged**